service.fussballpopup/menu.py

```diff
@@ -0,0 +1,310 @@
+
+profile    = xbmc.translatePath( __addon__.getAddonInfo('profile') ).decode("utf-8")
+temp       = xbmc.translatePath( os.path.join( profile, 'temp', '') ).decode("utf-8")
+translation = __addon__.getLocalizedString
+
+popupaddon=xbmcaddon.Addon("service.popwindow")
+popupprofile    = xbmc.translatePath( popupaddon.getAddonInfo('profile') ).decode("utf-8")
+popuptemp       = xbmc.translatePath( os.path.join( popupprofile, 'temp', '') ).decode("utf-8")
+  
+icon = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')+'/icon.png').decode('utf-8')
+
+defaultBackground = background
+defaultThumb = ""
+
+
+
+
+        
+def debug(content):
+    log(content, xbmc.LOGDEBUG)
+    
+def notice(content):
+    log(content, xbmc.LOGNOTICE)
+
+def log(msg, level=xbmc.LOGNOTICE):
+    addon = xbmcaddon.Addon()
+    addonID = addon.getAddonInfo('id')
+    xbmc.log('%s: %s' % (addonID, msg), level) 
+    
+    
+# Einlesen von Parametern, Notwendig für Reset der Twitter API
+def parameters_string_to_dict(parameters):
+  paramDict = {}
+  if parameters:
+    paramPairs = parameters[1:].split("&")
+    for paramsPair in paramPairs:
+      paramSplits = paramsPair.split('=')
+      if (len(paramSplits)) == 2:
+        paramDict[paramSplits[0]] = paramSplits[1]
+  return paramDict
+
+if not xbmcvfs.exists(temp):
+   xbmcvfs.mkdirs(temp)
+
+def ersetze(text):
+#
+    text=text.replace ("</p>","").replace("<p>","").replace("<div id='articleTranscript'>","").replace("<br />","").replace('<div id="image-caption">',"").replace("	","").replace("<p","")
+    text=text.replace ("<em>","").replace("</em>","")
+    text=text.replace ("<h3>","").replace("</h3>","")
+    text=text.replace ("<hr>","")
+    text = text.replace("&quot;", "\"")
+    text = text.replace("&apos;", "'")
+    text = text.replace("&amp;", "&")
+    text = text.replace("&lt;", "<")
+    text = text.replace("&gt;", ">")
+    text = text.replace("&laquo;", "<<")
+    text = text.replace("&raquo;", ">>")
+    text = text.replace("&#039;", "'")
+    text = text.replace("&#8220;", "\"")
+    text = text.replace("&#8221;", "\"")
+    text = text.replace("&#8211;", "-")
+    text = text.replace("&#8216;", "\'")
+    text = text.replace("&#8217;", "\'")
+    text = text.replace("&#9632;", "")
+    text = text.replace("&#8226;", "-")
+    text = text.replace('<span class="caps">', "")
+    text = text.replace('</span>', "")
+    text = text.replace('\u00fc', "ü")  
+    text = text.replace('\u00e4', "ä")     
+    text = text.replace('\u00df', "ß")      
+    text = text.replace('\u00f6', "ö")      
+    text = text.replace('\/', "/")    
+    #text = text.replace('\n', "")    
+    text = text.strip()
+    return text
+
+def addDir(name, url, mode, iconimage, desc="",sortname=""):
+    u = sys.argv[0]+"?url="+urllib.quote_plus(url)+"&mode="+str(mode)+"&name="+urllib.quote_plus(name)
+    ok = True
+    liz = xbmcgui.ListItem(name, iconImage=icon, thumbnailImage=iconimage)
```

Comment:
as usual, iconImage and thumbnailImage are deprecated, use setArt instead. Also, please make an effort to keep this in mind for future submissions :)